applications: serial_lte_modem: support A-GPS and P-GPS combined (#12359)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

  * An ``auto_connect`` operation in the ``#XCARRIER`` carrier command.
    The operation controls automatic registration of UE to LTE network.
  * ``#XNRFCLOUDPOS`` AT command to send location requests to nRF Cloud using cellular or Wi-Fi positioning, or both.

>   * Support to use A-GPS and P-GPS at the same time.

Reviewer: ```suggestion
  * Support for using A-GPS and P-GPS at the same time.
```

  * Operations to read or erase the MCUboot secondary slot from the ``#XFOTA`` AT command because the application update process overwrites the slot in any case.
  * AT commands ``#XCELLPOS`` and ``#XWIFIPOS``.
    They are replaced by the ``#XNRFCLOUDPOS`` command that allows to combine cellular and Wi-Fi data to determine the device location.

>   * AT commands ``#XAGPS`` and ``#XPGPS``.

Reviewer: ```suggestion
  * The AT commands ``#XAGPS`` and ``#XPGPS``.
```

>     Their functionality is merged into the ``#XGPS`` AT command that now allows to use A-GPS and P-GPS at the same time.

Reviewer: ```suggestion
    Their functionality is merged into the ``#XGPS`` AT command that now allows using A-GPS and P-GPS at the same time.
```
